src/bci_build/templates.py

```diff
@@ -72,6 +72,10 @@
 LABEL com.suse.release-stage="{{ image.release_stage }}"
 {%- endif %}
 # endlabelprefix
+{%- if image.dockerfile_from_line %}
+LABEL org.opencontainers.image.base.name="%BASE_REFNAME%"
+LABEL org.opencontainers.image.base.digest="%BASE_DIGEST%"
+{%- endif %}
```

Comment:
Any reason why you put it below the labelprefix section?

Reply:
the only reason is that I hate that label duplication for oci defined standard tags with passion. 

You can recursively look up the base.digests though. e.g. the pcp image which stacks on init-image (and base.digest refers to which one) and then you use the init-image's base.digest to look up the underlying base image. 

So I think reducing the number of pointless duplicated labels (given that these should be annotations to begin with) was the main driving force for me.